src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java

```diff
@@ -261,13 +261,52 @@ private void initializeDestinations(SipGenerationOptions options) {
                     return entry;
                 });
 
-                cdmToDestMapper.put(mapping, destEntry);
+                if (mapping.getId().contains(":")) {
+                    for (String cdmId : listCdmIdsByArchivalCollectionId(mapping.getId())) {
+                        cdmToDestMapper.put(cdmId, destEntry);
+                    }
+                } else {
+                    cdmToDestMapper.put(mapping.getId(), destEntry);
+                }
             }
         } catch (IOException e) {
             throw new MigrationException("Failed to load destination mappings", e);
         }
     }
 
+    /**
+     * @param id archival collection id
+     * @return list of CDM ids
+     */
+    private List<String> listCdmIdsByArchivalCollectionId(String id) {
+        String[] splitId = id.split(":");
+        String idField = splitId[0];
+        String idValue = splitId[1];
+
+        List<String> cdmIds = new ArrayList<>();
```

Comment:
We should add a check here to return the empty list if the `idValue` is blank, so that we don't end up matching every item that doesn't have an archival collection id (hopefully this shouldn't be able to happen after one of the other comments is addressed, but might as well be safe)